hoot-rnd/src/main/cpp/hoot/rnd/schema/ImplicitTagRawRulesDeriver.cpp

```diff
@@ -625,7 +625,7 @@ void ImplicitTagRawRulesDeriver::_sortByWord(boost::shared_ptr<QTemporaryFile> i
 
   //sort by word, then by tag
   const QString cmd =
-    "sort -t'\t' -k2,2 -k3,3 --parallel=" + QString::number(_sortParallelCount) + " " +
+    "sort -t$'\t' -k2,2 -k3,3 --parallel=" + QString::number(_sortParallelCount) + " " +
```

Comment:
I'd like a core develop to verify this change as well.  I've added @bwitham for his review.

Reply:
Looks fine. Thanks.

Reply:
Did this still fail last night?  I'm thinking that this might work better:
```
    "sort -t'\\t' -k2,2 -k3,3 --parallel=" + QString::number(_sortParallelCount) + " " +
```
where escaping the `\` to `\\` would actually push `\t` as text to the command and not a tab character.

Reply:
We think the failure may be a result from the time the nightly build is ran, because running the build in Jenkins yesterday with all parameters succeeded. Another full parameter build is in progress for confirmation. Before the change to **-t$'\t'** I encountered the same build error as the nigthly, by performing a manual build using vagrant. After the change, detailed in the commit, I no longer saw the error from a manual build. I think changing the original commit to **-t'\\t'** isn't the solution were looking for, because that seems to cause the sort issue with the misuse of shell built-ins, that I saw previously with -t'\t'. Also, the test files that are inputs are tab separated. I think changing the field-separator (-t flag for sort) from a tab to literal \t (\\t would escape the second \ making the -t flag pass the literal \t) would have the field separator evaluate the string \t instead of the tabs in the test files.